### DISCUSSION TITLE
If message x is a reply to message y, it should show the same in destination

### DISCUSSION THREAD

**Discussion by ehs4nm** [aahnik/tgcf#67] [Q&A] [answered]
when in a chat there is a reply to a former message, how we can manage to clone this behavior?
in live.py there should be defined a replied_message_handler() but how?

**Comment by aahnik** [comment, marked as answer]
**Update: The v0.2.3 release on 2 June 2021, brings reply sync to the table.** 

Check it out.

**Souce channel:**
![image](https://user-images.githubusercontent.com/66209958/120485120-654c1c00-c3d1-11eb-8652-ab9a798bb0b7.png)

-----

**Before v0.2.3**
![image](https://user-images.githubusercontent.com/66209958/120485399-aa704e00-c3d1-11eb-8a89-6e60817ac2ae.png)

**From v0.2.3 onwards**
![image](https://user-images.githubusercontent.com/66209958/120485098-5f563b00-c3d1-11eb-9e09-defa366df8de.png)

**Reply by kysterics** [reply]
Does this work for groups instead of channels?
I can't seem to get reply sync working...
I'm on 0.2.11.post0, Version 2.8.4

**Reply by tissole** [reply]
It should work everywhere. What is 2.8.4?

**Reply by kysterics** [reply]
> It should work everywhere. What is 2.8.4?

Version of Telegram.
